docs/source/user_guides.rst

```diff
@@ -55,13 +51,15 @@ User Guides
     	To know everything there is to know about a specific feature of XGI 
     	+++
 
-        .. button-ref:: indepth_tutorials
+        .. button-ref:: api/tutorials/in_depth
```

Comment:
Now you're using the path to the rst file rather than the anchor right? 
Is it better practice or what's idea behind it?

Reply:
Hard to know what best practices are. But, FWIW, matplotlib does it with relative paths: https://github.com/matplotlib/matplotlib/blob/main/doc%2Findex.rst

Reply:
Probably easier to track back the file it looks to this way